static/src/javascripts/lib/dom.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+const insertAfter = (
+    elementToInsert: Element,
+    elementToInsertAfter: Element
+): void => {
+    const parent = elementToInsertAfter.parentNode;
```

Comment:
These 10 lines become `elementToInsertAfter.insertAdjacentElement('afterend', elementToInsert)`. Same for the `insertBefore`: `elementToInsertBefore.insertAdjacentElement('beforebegin4', elementToInsert)`.